nservicebus/messaging/send-a-message.md

```diff
@@ -1,27 +1,43 @@
 ---
-title: Sending a Message
-summary: Describes how to send a message
+title: Sending messages
+summary: Describes how to send messages
 tags: []
 redirects:
 - nservicebus/how-do-i-send-a-message
 ---
 
-Message sending involves using Send operation that takes an argument of a message instance to be delivered. The details differ slightly between version of NServiceBus, as shown in following snippets.
+NServiceBus allows you to send different types of messages (e.g. Commands, Events, Replies, etc.) to itself or other endpoints. Messages can be sent from outside or inside the message processing pipeline:
 
-Here's how a message is sent by directly using the instance of the endpoint
+## Outside the pipeline
+
+Whenever you want to send a message which is not related to an incoming message, you're sending messages from outside the pipeline. Examples may be:
+* Subscribing to some events after starting up the bus.
```

Comment:
ok i am confused. do u mean an nservicebus message event? if so that would be inside the pipeline so why is it in this list